app/modules/edit_content.js

```diff
@@ -8,8 +8,8 @@ export default function editTask() {
       if (e.key === 'Enter') {
         e.preventDefault();
         editedText = todo.innerText;
+        console.log(editedText);
       }
-      return editedText;
     });
   });
```

Comment:
- [ ] As the requirements suggest, All changes to the To-Do List should be saved in local storage. When I reload the page, the edited task is discarded. Consider ensuring that the edited task is persistent(saved in localstorage).

Reply:
Thanks for the feedback.  I'm going to implement that.